### DISCUSSION TITLE
is it compatible with jellyseerr?

### DISCUSSION THREAD

**Discussion by killmetohell** [Feramance/qBitrr#122] [Q&A] [answered]
I notice it it says works with overseerr and i know from jellyseerr its fork from overseerr i was wondering if it compatible with jellyseerr or not.

**Comment by Feramance** [comment]
Hi, I honestly have no idea, never tested, try it out and let me know! Use the overseerr section ideally

**Comment by killmetohell** [comment]
it seem to work but not sure but it hasn't thrown errors about it i think.
To bad i don't know how to effectively to use qbitrr.

**Reply by Feramance** [reply, marked as answer]
I would assume it would work given that Jellyseerr is based on Overseerr though I wasn't sure. If you ever notice any errors please let me know and I'll take a look
